conf/modules.config

```diff
@@ -43,14 +58,21 @@ process {
         ]
     }
 
-
-    withName: VCF2MAF {       
-        ext.args =  { $params.run_vep ? "" : "--inhibit-vep" }  
+    withName: VCF2MAF {
+        ext.args = { [ "",
+            params.run_vep ? "" : "--inhibit-vep",
+            meta.normal_id ? "--normal-id ${meta.normal_id} --vcf-normal-id ${meta.vcf_normal_id}" : "",
```

Comment:
I take it if a `normal_id` is present in the meta, it is ensured  that a `vcf_normal_id` will _always_ be present as well?

Reply:
yep